### DISCUSSION TITLE
Cockpit instrument light dimming

### DISCUSSION THREAD

**Discussion by aeroniemi** [magknight/787aviatorsTracker#529] [Completed things] [answered]
Would require sim/graphics/misc/light_attenuation
Dimming some (all?) cockpit instrument lights by button during night

**Comment by aeroniemi** [comment]
indicators are auto-dimmed, rolling out across the cockpit currently

**Comment by aeroniemi** [comment, marked as answer]
Rollout completed in 1.7.5-alpha7, will go to live with rest of cygnus branch

**Comment by KBNOtaku** [comment]
but now all the button lights are competly dark at night? 1.7.5-alpha13
![B7879 - 2021-05-28 13 43 16](https://user-images.githubusercontent.com/56799893/120062104-9985ac80-c060-11eb-86ca-6123d5bfe423.png)

**Reply by aeroniemi** [reply]
i can't replicate that happening - it works fine for me at night!
![image](https://user-images.githubusercontent.com/13006900/120087703-1f960780-c0e2-11eb-8eb9-4b23c2f85829.png)
